django_celery_results/models.py

```diff
@@ -137,15 +138,71 @@ class ChordCounter(models.Model):
 
     def group_result(self, app=None):
         """Return the GroupResult of self.
-
         Arguments:
         ---------
             app (Celery): app instance to create the GroupResult with.
-
         """
         return GroupResult(
             self.group_id,
             [result_from_tuple(r, app=app)
              for r in json.loads(self.sub_tasks)],
-            app=app,
         )
+
+
+class GroupResult(models.Model):
```

Comment:
This name clashes with `from celery.result import GroupResult`

Reply:
I agree that it does.. however [in the celery docs](https://docs.celeryproject.org/en/latest/faq.html?highlight=TaskResult#what-purpose-does-the-database-tables-created-by-django-celery-results-have) they refer to a GroupResult model in this project, even though it doesnt exist yet (maybe it did at some point?)

[this fork](https://github.com/celery/django-celery-results/compare/master...CantemoInternal:cantemo) of django-celery-results used `TaskSetMeta` for the model name. It seems like `celery.result.GroupResult` used to be called `TaskSetResult`, so there was probably a collision there as well.


if the concern is with collisions specifically in the `models.py` file, we could use the full path `celery.result.GroupResult` where the actual celery class is needed.  But maybe its better to avoid collisions altogether

Reply:
I've switched to more explicit imports in `models.py` so at least we dont have name clashes in the file